text/0000-rebalancing-coherence.md

```diff
@@ -0,0 +1,293 @@
+- Feature Name: fundamental_attribute
+- Start Date: 2015-03-27
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+## Summary
+
+This RFC proposes two rule changes:
+
+1. Modify the orphan rules so that impls of remote traits require a
+   local type that is either a struct/enum/trait defined in the
```

Comment:
`type` and `use` introduce pure aliases and are irrelevant here. This is focusing on the core type system, in which no aliases exist.